tensorflow/core/kernels/BUILD

```diff
@@ -538,6 +538,11 @@ cc_library(
     hdrs = ["conv_ops_gpu.h"],
 )
 
+cc_library(
+    name = "pooling_ops_gpu_hdrs",
+    hdrs = ["maxpooling_op_gpu.h"],
+)
+
```

Comment:
Is this necessary? I don't see it used anywhere. Please remove?

Reply:
I will follow up

Reply:
@penpornk  Thank you very much for catching the problem. I missed loading change in kernes/mkl/BUILD file, which is needed for GPU tests of quantized pooling ops.  Fix has been done.